src/frontend/src/features/layouts/components/thread-view/components/attachment-preview-modal/_index.scss

```
@@ -0,0 +1,163 @@
+    flex-direction: column;
+    gap: var(--c--globals--spacings--4xs);
+}
+
+.attachment-preview-sidebar__field-label {
+    font-size: var(--c--globals--font--sizes--xs);
+    color: var(--c--contextuals--content--semantic--neutral--secondary);
+}
+
+.attachment-preview-sidebar__field-value {
+    margin: 0;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)